graylog2-web-interface/src/components/event-definitions/event-definition-types/FilterForm.jsx

```diff
@@ -231,6 +226,14 @@ class FilterForm extends React.Component {
       return onChange('config', newConfig);
     };
 
+    if (!this._userCanViewLookupTables()) {
+      return (
+        <Alert bsStyle="info">
+          Only Admins are able to declare Query Parameters from Lookup Tables.
```

Comment:
```suggestion
          This account lacks permission to declare Query Parameters from Lookup Tables.
```
I think we should phrase this differently. If, for example we decide to give the alerts manager role the lookuptable:read permission.